### PR TITLE
QnA Maker documentation generation - xml

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

File: sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj [hunk 1]
```diff
@@ -20,4 +20,8 @@
     <AzureApiTag />
     <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
   </PropertyGroup>
+
+  <PropertyGroup >
+    <DocumentationFile>$(IntermediateOutputPath)$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj]: @chidozieononiwu is this needed or we have some other way to pull this in?

Comment (chidozieononiwu) [reply sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj]: Yes it is required. Although it is being pulled in as part of the build [here](https://github.com/Azure/azure-sdk-for-net/blob/4c390c8bdb5fe020e7890a7d5129e7222db965fc/eng/Directory.Build.Data.props#L118). Right now it is conditioned for only Track 2 libraries. Track 1 libraries have been adding the DocumentationFile seperately.

**Review by AlexGhiondea — APPROVED** [review]

**Comment by AlexGhiondea** [comment]
@diberry could you look at the failures [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=337266&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=3c663f50-2031-5e5a-5a27-a536dda269a8).

It looks like there are members that missing XML comments.

**Comment by jsquire** [comment]
This request hasn't had forward progress in several months and appears to be stale.   I'm going to close this out; @diberry, please feel free to reopen if you'd like to pursue these changes.

**Pull request closed without merging**